apps/meteor/client/views/room/composer/ComposerOmnichannel/ComposerOmnichannelCallout.tsx

```diff
@@ -53,20 +46,11 @@ const ComposerOmnichannelCallout = () => {
 					<Button danger secondary small onClick={handleBlock}>
 						{currentChannel?.blocked ? t('Unblock') : t('Block')}
 					</Button>
+					<IconButton icon='cross' secondary small title={t('Close')} onClick={() => setDismissed(true)} />
 				</ButtonGroup>
 			}
 		>
-			{shouldShowSecurityRoute ? (
```

Comment:
why we won't display anymore the option to enable the verification feature?

Reply:
Even though we had the appropriate permission checks, the link was mostly relevant to admins/managers. 
The goal is to reduce the callout information to what's relevant within the conversation context, while activation/setup details will be handled in a future recommended actions page.